cocos/ui/UIEditBox/Mac/CCUIEditBoxMac.mm

```diff
@@ -135,6 +135,8 @@ - (NSWindow *)window
 - (void)openKeyboard
 {
     [self.window.contentView addSubview:self.textInput];
+    //FIXME: it will cause a few OpenGL error once, should be fix when glfw version upgraded.
```

Comment:
There is a few OpenGL errors, but it doesn't affect the rendering and these error only occurs once. I think the problem should be resolved in the glfw upstream. But I can't handle it properly at the moment. This is a workaround, but at least it works well except for the error message.